src/components/AgentRow.tsx

```diff
@@ -68,8 +80,9 @@ const AgentRow: React.FC<AgentRowProps> = (
   const children = getChildren(agent);
 
   return (
-    <div style={{ display: 'flex', flexDirection: 'column', width: '100%' }}>
-      <div className="item-row" style={{ display: 'flex' }}>
+    <div>
+      <div className="item-row">
+        {hasChildren ? Caret : Spacer}
```

Comment:
I don't expect these names to be capitalized because they aren't types.  Is there a naming convention about react component instances that I'm missing?  I would expect capitalized ones to be inside of html element angle brackets.

Reply:
Definitely not a React convention! This pattern exists in the website but I like changing the declarations to functions so that the use is like `<Caret />` instead of `Caret` if its clearer to read.

Reply:
`<Caret/>` would have been more correct capitalization IMO

Reply:
just pushed this change :)